README.md

```diff
@@ -9,7 +9,90 @@ Welcome to the official repository for the extended abstracts of the Digital His
 
 ## Introduction
 
-This repository hosts the standardized template for submitting extended abstracts to the Digital History Switzerland 2024 conference. It's designed to ensure uniformity in submissions, making the review process more efficient and accessible.
+This repository hosts the standardized template for submitting extended abstracts to the Digital History Switzerland 2024 conference. It's designed to ensure uniformity in submissions, making the review process more efficient and accessible. And it looks nice, too.
+
+![Preview of the abstract template.](screenshot.png)
+
+### Getting Started
+
+To get started with the abstract template, follow these steps:
+
+1. **Create a GitHub Account:**
+   - If you do not already have a GitHub account, create one [here](https://github.com/signup) for free.
+
+2. **Create Your Repository:**
+   - Click the ["Use this template"](https://github.com/new?template_name=abstract-template&template_owner=digihistch24) button to start a new repository based on this template.
+   - Enter a name for your repository and decide whether to make it public or private.
+   - Click "Create repository from template" to set up your new project.
+
+3. **Access and Edit Your Abstract:**
+   - In your new repository, navigate to the `abstract-template` folder.
+   - Find the `index.qmd` file and click on it.
+   - Click the pencil icon (edit) in the top right corner of the file view to begin editing.
+
+4. **Commit Your Changes:**
+   - After making your edits, scroll to the bottom of the page.
+   - Enter a brief description of the changes you made in the "Commit changes" box.
+   - Click "Commit changes" to save your edits.
+
+5. **Preview and Collaborate:**
+   - To preview your abstract, follow the steps in the "Previewing the Abstract" section below.
+   - To collaborate, go to the "Settings" tab in your repository, choose "Manage access," and then "Invite a collaborator."
+
+6. **Submit Your Abstract:**
+   - When your abstract is ready for submission, download the repository as a ZIP file.
```

Comment:
Do we want the zip file as standard/preferred mode of submission? I think we should only suggest this as an alternative method. Won't we just end up with lots of emails with zip folders when somebody decides to update something afterwards?